packages/block-library/src/search/index.php

```diff
@@ -194,15 +195,25 @@ function styles_for_block_core_search( $attributes ) {
 		);
 	}
 
+	// Add border width styles.
+	$has_border_width = ! empty( $attributes['style']['border']['width'] );
+	$border_width     = $attributes['style']['border']['width'];
```

Comment:
To avoid a PHP undefined index notice when no width is set, I think we need to move this line to beneath the `if ( $has_border_width )` check two lines below.

![image](https://user-images.githubusercontent.com/14988353/145324015-b4d2767c-bf41-4fb7-9cf8-e9c42ea54211.png)

Reply:
Thanks again! I think I always forget to add this 🤣